app/app.py

```diff
@@ -109,10 +109,26 @@ def find_recipe_in_url_window_algo_based():
 
     ingred_paragraph, instr_paragraph = extract(ingredients, instructions, url)
 
+    # TODO - use remove_unwanted_patterns - if we want to support english also
+    # ingred_paragraph = remove_unwanted_patterns(ingred_paragraph)
+    # instr_paragraph = remove_unwanted_patterns(instr_paragraph)
+
     return {'ingredients': ingred_paragraph,
             'instructions': instr_paragraph}
 
 
+
+def remove_unwanted_patterns(text):
+    # Regex for removing all urls and file path:
+    # First group is for matching file path
+    # After the boolean or '|' groups that match all URI
+    url_regex = r'([\w]+(\/.*?\.[\w:]+))|([\w+]+\:\/\/)?([\w\d-]+\.)*[\w-]+[\.\:]\w+([\/\?\=\&\#.]?[-\w\+\,\%\=\'\"\:\/]+)*\/?'
```

Comment:
two improving options:
or make it constant so it wont be calculated every time we access it.
or put it in some file so we can modified the regex easily